### PR TITLE
Deploy Ada on Kubernetes

### PR DESCRIPTION
**Pull request by bmarkons**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:8.6-slim
+
+RUN apt-get update && apt-get -y install redis-server
```
Comment (shiroyasha) [inline Dockerfile]: Redis should be a separate docker image.

Comment (bmarkons) [reply Dockerfile]: These two should be in the same Pod then?

Comment (shiroyasha) [reply Dockerfile]: No. Two separate pods.

On your local machine, you will need to start up a Redis docker image.
In production, we will connect to a hosted Redis solution.

We don't want to run databases, cache-servers, or any other image that has permanent storage in the kubernetes cluster in production. We will rely on managed, hosted solutions.

But, for local testing we need to start up a seperate redis image to simulate the hosted service.

**Comment by bmarkons** [comment]
I've made separate minikube.yml file for deploying on local minikube cluster. Also added REDIS_URL env variable in deploy.yml which would point to our hosted Redis server.

The next step is to setup deploy commands on Semaphore to push image to Dockerhub. After which we can play on Google Cloud.

WDYT? @shiroyasha

**Comment by shiroyasha** [comment]
Sounds great. 👍 Godspeed.

**Comment by shiroyasha** [comment]
Please update the readme before merging. :)

**Comment by bmarkons** [comment]
Ack :trollface:

**Pull request merged**